.github/ISSUE_TEMPLATE/ios_background_message_handling.yml

```diff
@@ -0,0 +1,166 @@
+  Create an issue after ensuring you have understood iOS background message handling, its limitations and the required configuration.
+title: "fix:  "
+labels: ["bug", "triage"]
+# assignees:
+#   - ...
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: |
+        Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues.
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: Are you aware of the fundamental differences between iOS and Android background message handling?
+      description: |
+        iOS adopts a more aggressive policy to throttling background messages. Android is a lot more lenient, even when the app is in a terminated state, it is highly likely that
+        messages will be received by the device and processed by the Firebase Messaging background handler. This is not the case with iOS. There are multiple reasons why your messages
+        may not be received by the device. Here are just a few reasons that are weighed by the iOS system in the decision to throttle your messages:
+        1. Low battery
+        2. High CPU usage
+        3. Too many messages received
+        4. App is in a terminated or background state
+        5. Data-only messages are considered lower priority than notification messages
+      options:
+        - label: I understand that iOS and android background messages behaves differently, and I've designed my application with that in mind.
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: Do you have an active Apple Developer account?
+      description: |
+        An [Apple Developer account](https://developer.apple.com/membercenter/index.action) is required.
+      options:
+        - label: I have an active Apple Developer account.
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: Are you using a physical iOS device to test background messages?
+      description: |
+        A physical device is required, background messages will not work on a simulator.
+      options:
+        - label: I am using a physical iOS device to test background messages.
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Have you enabled "Remote Notifications" & "Background Mode" (Checking options for "Background Processing" & "Remote Notifications") in your app's Xcode project?
+      description: |
+        Find out how to enable "Remote Notifications" & "Background Mode" in the [documentation](https://firebase.google.com/docs/cloud-messaging/flutter/client#enable_app_capabilities_in_xcode).
+
+        For illustration, your Xcode project "Signing and Capabilities" settings should look like this:
+        ![Xcode "Signing and Capabilities tab"](https://github.com/firebase/flutterfire/blob/master/.github/images/remote-notifications-enable.png)
```

Comment:
```suggestion
        ![Xcode "Signing and Capabilities tab"](/.github/images/remote-notifications-enable.png)
```

Testing - I think you can just put a relative branch rather than a full url